### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by jpearson-sfdc**
Fixing broken links in multiple pages.

Also removing querying-visits.md pages in both versions since they aren't linked to and the content appears to be a duplicate of text in visits.md.

### CODE REVIEW

File: docs/kb/api-version-3/export.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ Output Representation
 ### Errors
 
 * Status Code: 4xx
-* Error codes: See [Error Codes](/kb/error-codes-messages)
+* Error codes: See [Error Codes](../error-codes-messages.md)
```
Comment (franciscosanchezsfdc) [inline docs/kb/api-version-3/export.md]: The error codes link seems to work already. This change may not be needed. Also the extra *.md may generate an invalid link

Comment (jpearson-sfdc) [reply docs/kb/api-version-3/export.md]: Just the opposite; the link with the ".md" is proper way to reference links to other pages within the site and the "/kb/" link may break in the future.

File: docs/kb/api-version-3/querying-visits.md [hunk 2]
```diff
@@ -1,53 +0,0 @@
-
```
Comment (adelawalla) [inline docs/kb/api-version-3/querying-visits.md]: Is this file not used?

Comment (jpearson-sfdc) [reply docs/kb/api-version-3/querying-visits.md]: They are not linked from anywhere. The querying-visits.md files appear to be old files and the content is duplicated in visits.md.

**Pull request merged**